test/cases/api.sh

```diff
@@ -703,8 +736,10 @@ sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SEL
 # Save the Manifest from the osbuild-composer store
 # NOTE: The rest of the job data can contain sensitive information
 #
-# Suppressing shellcheck.  See https://github.com/koalaman/shellcheck/wiki/SC2024#exceptions
-sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SELECT result->>'Manifest' FROM jobs" | sudo tee "${ARTIFACTS}/manifest.json"
+# Disable -x for this command to avoid spewing the whole result and manifest into the log
+set +x
+sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SELECT result FROM jobs WHERE type='manifest-id-only'" | jq .data | sudo tee "${ARTIFACTS}/manifest.json"
```

Comment:
straight up piping to `jq` won't work because you have all the SQL stuff around it.

Maybe 

`SELECT result->>'data' FROM jobs WHERE type='manifest-id-only' | sudo tee.. `

It still won't b proper json though; or maybe using `psql -t` ; `-t` is supposed to surpress the psql header/footer.

Reply:
yeah, that was a bit silly.  I decided instead to just dump the result into the artifacts.